source/codegen/metadata/nidmm/config.py

```diff
@@ -1,29 +1,16 @@
-﻿# -*- coding: utf-8 -*-
-# This file is generated from NI-DMM API metadata version 19.6.0d7
+# -*- coding: utf-8 -*-
+# This file is generated from NI-DMM API metadata version 22.8.0d9999
 config = {
     'additional_headers': {
-        'custom/ivi_errors.h': [
-            'service.cpp'
-        ]
     },
-    'api_version': '19.6.0d7',
+    'api_version': '22.8.0d9999',
     'c_function_prefix': 'niDMM_',
     'c_header': 'nidmm.h',
     'close_function': 'Close',
-    'context_manager_name': {
-        'abort_function': 'Abort',
-        'initiate_function': 'Initiate',
-        'task': 'acquisition'
-    },
     'csharp_namespace': 'NationalInstruments.Grpc.Dmm',
     'custom_types': [
     ],
     'driver_name': 'NI-DMM',
-    'extra_errors_used': [
-        'InvalidRepeatedCapabilityError',
-        'SelfTestError'
-    ],
-    'init_function': 'InitWithOptions',
```

Comment:
init_function wasn't actually used. The init methods are specified with `'init_method': True` in functions.py